platform/commonUI/browse/bundle.json

```diff
@@ -92,8 +92,8 @@
             },
             {
                 "key": "window",
-                "implementation": "windowing/NewWindowAction.js",
-                "description": "Open this object in a new window",
+                "implementation": "windowing/NewTabAction.js",
+                "description": "Open this object in a new tab",
                 "category": "view-control",
```

Comment:
We should add "contextual" as a category here (e.g. `"category": [ "view-control", "contextual" ]`) so that this appears in the context menu of a domain object on right-click

Also want to give it a `"name": "Open in New Tab"` so that this has a name in the context menu.